code/game/objects/structures/window.dm

```diff
@@ -333,6 +333,13 @@
 	set_dir(turn(dir, 90))
 	update_nearby_tiles(need_rebuild=1)
 
+/obj/structure/window/set_dir(ndir)
+	..()
```

Comment:
`. = ..()`; it returns true if dir was changed, I believe

Reply:
Fixed